signetdev/host/signetdev_linux.c

```diff
@@ -63,7 +63,9 @@ void signetdev_priv_handle_error()
 static void command_response(int rc)
 {
 	char resp = rc;
-	write(g_command_resp_pipe[1], &resp, 1);
+	if (write(g_command_resp_pipe[1], &resp, 1) == -1) {
+		perror("command_response write error");
```

Comment:
This is too low level of a function to be printing. It should be returning a value to the caller. The callers of command_response() can just use __attribute__((unused)) with the return value at first as long as there is a "//TODO" comment for each one to properly catch errors in the end

Reply:
sure, that, can be done.